### PR TITLE
Species based min and max age

### PR DESCRIPTION
**Pull request by LittleBigKid2000**
Machines, Grey, and Dionae can now be anywhere from 0 to 150 years old. All other species still have the normal 17 to 85 restriction.
Ages are rounded down to 85 and up to 17 when calculating the pitch of screams. In case I just completely failed to write a coherent sentence, a 1-16 year old person will have the same scream as a 17 year old, and a 86-150 year old will have the same scream as an 85 year old person.
I would like for min and max age to be depended on something like a var in /datum/species, but that doesn't seem to be possible.

:cl:
tweak: IPCs, Grey, and Dionae can now be between 0-150 years old.
/:cl:

### CODE REVIEW

File: code/__DEFINES/mob.dm [hunk 1]
```diff
@@ -21,6 +21,8 @@
 #define AGE_MIN 17			//youngest a character can be
 #define AGE_MAX 85			//oldest a character can be
 
+#define AGE_MIN_E 0			//youngest a character with no age restriction (i.e. dionae, machine people, and grey) can be
```
Comment (tigercat2000) [inline code/__DEFINES/mob.dm]: what even
no

follow bay's example if you are going to do this at all, put a var/min_age and var/max_age on species

**Comment by tigercat2000** [comment]
6 commits for a 13 line change
:no_entry_sign:

File: code/__DEFINES/mob.dm (hunk 1 above)
Comment (LittleBigKid2000) [inline code/__DEFINES/mob.dm]: As I said, I couldn't find a way of making that work. I'll search harder.

**Comment by theColdflame** [comment]
Haven't IPCs only existed for ~50 years?

**Comment by LittleBigKid2000** [comment]
Yeah, I'm redoing it to be based on a var in species, if I can figure out how.

**Comment by LittleBigKid2000** [comment]
Fuck it, someone else can do it. I feel like I'm going to have a brain aneurysm

**Comment by PJB3005** [comment]
> 6 commits for a 13 line change

It's called squash merge.

**Pull request closed without merging**